giraffe/src/components/Tooltip.tsx

```diff
@@ -40,9 +41,14 @@ export const Tooltip: FunctionComponent<Props> = ({data, config}) => {
     return TOOLTIP_MAXIMUM_OPACITY
   }, [legendOpacity])
 
-  const columns = columnsWhitelist
-    ? data.filter(column => columnsWhitelist.includes(column.key))
-    : data
+  let columns = []
+  if (disableTooltip != true) {
```

Comment:
Nitpick: we should use `!==` here. If `disableTooltip` is the value `1` or `"1"` the condition becomes false and the tooltip will be disabled. TypeScript helps here. Being more explicit is good too.

Reply:
Done! Explicit is good!